x-pack/plugins/cases/public/components/all_cases/expanded_row.tsx

```diff
@@ -6,15 +6,15 @@
  */
 
 import React from 'react';
-import { EuiBasicTable as _EuiBasicTable } from '@elastic/eui';
+import { EuiBasicTable } from '@elastic/eui';
 import styled from 'styled-components';
 import { Case, SubCase } from '../../containers/types';
 import { CasesColumns } from './columns';
 import { AssociationType } from '../../../common';
 
 type ExpandedRowMap = Record<string, Element> | {};
 
-const EuiBasicTable: any = _EuiBasicTable;
+// @ts-expect-error TS2769
```

Comment:
I could disable the linting rule about `any` but I found it better to use `@ts-expect-error` because we will be warned if it will be fixed and we do not have to declare `_EuiBasicTable` (better readability)